site/content/user-guide/amqp-adapter.md

```diff
@@ -51,13 +64,13 @@ To run the client using the above default values, open a terminal and execute th
 
     $ /hono/cli/target$ java -jar hono-cli-0.7-SNAPSHOT-exec.jar --spring.profiles.active=amqp-adapter-cli
 
-To run the client with a different value for the message address, username, password and payload body, we supply those values to command by doing the following:
+To run the client with a different value for the message address, username, password and payload body, we supply those values to the command by doing the following:
```

Comment:
please do not use first person (*we supply*)

Reply:
ok